Description: Fix cme shebang
 Debian Policy ยง 10.4 states that Perl scripts must use /usr/bin/perl directly and not /usr/bin/env
Author: dod
Applied-Upstream: NA
--- a/bin/cme
+++ b/bin/cme
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+#!/usr/bin/perl
 #
 # This file is part of App-Cme
 #
